File: src/api/health.ts
import client from './client'

export interface HealthStatus {
  status: string
  version: string
  python_version: string
}

export default {
  /** 健康检查 */
  get(): Promise<HealthStatus> {
    return client.get('/health').then((r) => r.data)
  },
}

File: server/routers/health.py
"""健康检查端点"""

from __future__ import annotations

import sys

from fastapi import APIRouter

from .. import dependencies

router = APIRouter()


@router.get("/health")
async def health() -> dict[str, str]:
    return {
        "status": "ok",
        "version": dependencies.get_app_version(),
        "python_version": sys.version,
    }
